linetools/lists/sets/llist_v1.1.ascii

```diff
@@ -1121,7 +1121,7 @@
 | 1200.7098 |    NI 1200c |    1 |   1 |   0 |   0 |   0 |    0 |   0 |   0 |    0 |
 |    1206.5 |  SiIII 1206 |    1 |   1 |   0 |   0 |   0 |    0 |   0 |   0 |    0 |
 | 1215.3394 |     DI 1215 |    1 |   0 |   0 |   0 |   0 |    0 |   0 |   0 |    0 |
-|   1215.67 |     HI 1215 |    1 |   1 |   1 |   0 |   0 |    0 |   0 |   0 |    0 |
+|   1215.67 |     HI 1215 |    1 |   1 |   1 |   0 |   0 |    0 |   1 |   0 |    0 |
```

Comment:
That column is for galaxies.  And specifically emission.
No Lyb is expected (or detected to date).